sdk/storage/storage-blob/test/perfstress/track-1/storageTest.spec.ts

```diff
@@ -0,0 +1,60 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { generateUuid } from "@azure/core-http";
+import {
+  Aborter,
+  ContainerURL,
+  ServiceURL,
+  SharedKeyCredential,
+  StorageURL
+} from "@azure/storage-blob";
+import { PerfStressTest, getEnvVar } from "@azure/test-utils-perfstress";
+
+// Expects the .env file at the same level as the "test" folder
+import * as dotenv from "dotenv";
+dotenv.config({ path: "../../../.env" });
+
+export abstract class StorageBlobTest<TOptions> extends PerfStressTest<TOptions> {
+  blobServiceClient: ServiceURL;
+  containerClient: ContainerURL;
+  static containerName = generateUuid();
+
+  constructor() {
+    super();
+    const connectionString = getEnvVar("STORAGE_CONNECTION_STRING");
+    const accountName = getValueInConnString(connectionString, "AccountName");
+    const accountKey = getValueInConnString(connectionString, "AccountKey");
+
+    const sharedKeyCredential = new SharedKeyCredential(accountName, accountKey);
+    this.blobServiceClient = new ServiceURL(
+      `https://${accountName}.blob.core.windows.net`,
+      StorageURL.newPipeline(sharedKeyCredential)
+    );
+    this.containerClient = ContainerURL.fromServiceURL(
+      this.blobServiceClient,
+      StorageBlobTest.containerName
+    );
+  }
+
+  public async globalSetup() {
+    await this.containerClient.create(Aborter.none);
+  }
+
+  public async globalCleanup() {
+    await this.containerClient.delete(Aborter.none);
+  }
+}
+
+export function getValueInConnString(
```

Comment:
Minor: I prefer full name
```suggestion
export function getValueInConnectionString(
```